<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_circle</class>
 <widget class="QWidget" name="Page_circle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>大半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>大半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="circle_image_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="circle_image_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>54</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="circle_lineE_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
